### PR TITLE
Update native-classes-in-depth.md

### PR DESCRIPTION
**Pull request by nruth**
I found alert confusing, it's not defined & console.log in the caller would expect a string return value?

### CODE REVIEW

**Review by locks — APPROVED** [review]
Thanks!

File: guides/release/in-depth-topics/native-classes-in-depth.md [hunk 1]
```diff
@@ -426,7 +426,7 @@ directly on the class itself.
 ```js
 class Alert {
   static helloWorld() {
-    alert('Hello, world!');
```
Comment (ijlee2) [inline guides/release/in-depth-topics/native-classes-in-depth.md:429]: I believe the `alert` refers to the built-in `Window.alert()` method. https://developer.mozilla.org/docs/Web/API/Window/alert

Based on lines 433 and 437, I think you are correct that `helloWorld` should be returning a string.

**Review by ijlee2 — APPROVED** [review]

**Comment by ijlee2** [comment]
@nruth Would you like to make another pull request, to help out with backporting (copy-pasting) the fix to older versions of Ember Guides (versions 3.15 - 3.21)?

You can find the corresponding `native-classes-in-depth.md` file in the following folders:

- `guides/v3.15.0`
- `guides/v3.16.0`
...
- `guides/v3.21.0`

File: guides/release/in-depth-topics/native-classes-in-depth.md (hunk 1 above)
Comment (nruth) [reply guides/release/in-depth-topics/native-classes-in-depth.md:429]: That sounds right. I wasn't sure if that returns a string though, and there's no return statement so I thought maybe someone started off the example with an alert pop-up then decided to use console.log instead, and forgot to switch to returning the string.

**Comment by nruth** [comment]
Sure, I'll give that a go now. If I see #1614 there too can I do both in one commit or is it best to keep them split up?

**Comment by ijlee2** [comment]
> Sure, I'll give that a go now. If I see #1614 there too can I do both in one commit or is it best to keep them split up?

Thanks in advance for your help! Since it's the same file with not many changes, I think 1 pull request is perfect. If you'd like, you can make 2 commits, one for each different type of changes made.

**Pull request merged**